demos/GFM/download_gfm_python.ipynb

```diff
@@ -11,58 +11,96 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this notebook, we demonstrate how to query the GFM STAC collection and\n",
-    "download the found results using the Python library [pystac_client](https://pystac-client.readthedocs.io/en/latest/index.html) together \n",
-    "with a time and area of interest. \n",
+    "EODC catalogs several datasets using the [STAC](http://stacspec.org/)\n",
+    "(SpatioTemporal Asset Catalog) specification. By providing a [STAC API](https://stac.eodc.eu/api/v1) endpoint,\n",
+    "we enable users to search our datasets by space, time and more filter criterias\n",
+    "depending on the individual dataset.\n",
     "\n",
-    "In the STAC items a link to the respective asset (=file) is saved. This link is\n",
-    "used to download the file to a specified folder on the local machine."
+    "In this notebook, we demonstrate how to query the [Global Flood Monitoring](https://extwiki.eodc.eu/en/GFM) (GFM) STAC collection using the Python library\n",
+    "[pystac_client](https://pystac-client.readthedocs.io/en/latest/index.html) and\n",
+    "download the data using built-in Python libraries as well as utilizing the command line tool [stac-asset](https://github.com/stac-utils/stac-asset).\n",
+    "\n",
+    "You can install the pystac_client via pip:\n",
+    "\n",
+    "    pip install pystac_client\n",
+    "\n",
+    "In the STAC items the respective assets (=file) are linked. These links are\n",
+    "used to download the file to a specified folder on your machine."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 16,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Some needed imports\n",
-    "import os\n",
-    "import urllib\n",
-    "import rich.table\n",
-    "from datetime import datetime\n",
     "from pystac_client import Client\n",
-    "from rich.console import Console\n",
-    "from shapely.geometry import box\n",
     "\n",
-    "console = Console()"
+    "# EODC STAC API URL\n",
+    "api_url = \"https://stac.eodc.eu/api/v1\"\n",
+    "\n",
+    "eodc_catalog = Client.open(api_url)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Specify search query"
+    "## Searching\n",
+    "\n",
+    "We can use the STAC API to find items that match specific criteria. This may\n",
+    "include the date and time the item covers, its spatial extent, or any other\n",
+    "property saved in the item's metadata.\n",
+    "\n",
+    "In this example we are searching for GFM data which cover our area of interest\n",
+    "over South Pakistan in September 2022.\n",
+    "\n",
+    "The area of interest can be specified as `bbox` using the Python library\n",
+    "`shapely` or, alternateively, as GeoJSON object.\n",
```

Comment:
Just one note here: Afaik, you can also query with just a list instead of a shapely bbox.